src/ConstructiveSolidGeometry/SurfacePrimitives/EllipticalSurface.jl

```diff
@@ -69,9 +69,10 @@ const PartialCircularArea{T} = EllipticalSurface{T,T,T}
 const Annulus{T} = EllipticalSurface{T,Tuple{T,T},Nothing}
 const PartialAnnulus{T} = EllipticalSurface{T,Tuple{T,T},T}
 
-Plane(es::EllipticalSurface{T}) where {T} = Plane{T}(es.origin, es.rotation * CartesianVector{T}(zero(T),zero(T),one(T)))
+Plane(es::EllipticalSurface{T}) where {T} = Plane{T}(es.origin, normal(es))
 
-normal(es::EllipticalSurface{T}, ::CartesianPoint{T} = zero(CartesianPoint{T})) where {T} = es.rotation * CartesianVector{T}(zero(T), zero(T), one(T))
+normal(es::EllipticalSurface{T}, ::CartesianPoint{T} = zero(CartesianPoint{T})) where {T} = 
+    _transform_into_global_coordinate_system(CartesianVector{T}(zero(T), zero(T), one(T)), es)
```

Comment:
Or here: we rotate `[0,0,1]` into the global coordinate system using the `rotation` of the primitive, in this case the `EllipticalSurface`